### PR TITLE
dasd probing enhancements

### PR DESCRIPTION
**Pull request by mwhudson**
* Report more information (including ECKD vs FBA) about "real" dasds
 * Include dasds-passed-via-virtio in the set of dasds.

### CODE REVIEW

**Comment by mwhudson** [comment]
I'll add dasdview output for a FBA drive before merging, but the s390x systems are currently inaccessible due to the DC move

**Comment by mwhudson** [comment]
@raharper ping?

File: probert/tests/test_dasd.py [hunk 1]
```diff
@@ -7,6 +7,36 @@
...
+        'cylinders': 30051,
+        'device_id': device_id,
+        'disk_layout': 'cdl',
+        'name': devname,
+        'tracks_per_cylinder': 15,
+        'type': 'ECKD',
+        }
+
+
+def expected_dasde_probe_data(*, device_id):
```
Comment (raharper) [inline probert/tests/test_dasd.py:28]: I do find dasdd and dasde to be hard to read embedded in that function name; almost like a typo.

```
expected_probe_data = {
   'dasdd': {...},
   'dasde': {...},
}

self.assertEqual(expected_probe_data['dasdd'], ...)

```

**Review by raharper — APPROVED** [review]
Looks good.  One minor preference around readability of names, but it's fine to go in as it is.

File: probert/tests/test_dasd.py (hunk 1 above)
Comment (mwhudson) [reply probert/tests/test_dasd.py]: Yeah that makes sense.

**Pull request merged**